restli-server/src/main/java/com/linkedin/restli/server/config/RestLiMethodConfigImpl.java

```diff
@@ -24,7 +26,8 @@ public boolean shouldValidateQueryParams() {
     return _validateQueryParams;
   }
 
-  public void setValidateQueryParams(boolean validateQueryParams) {
-    _validateQueryParams = validateQueryParams;
+  @Override
+  public boolean shouldValidateResourceKeyParams() {
```

Comment:
nit: If you agree with my other suggestion, then you should also rename this to `#shouldValidateResourceKeys` (and the variable to `_validateResourceKeys`) to be consistent.

Reply:
refer reply on other conversation